app/views/admin/users/index.html.slim

```diff
@@ -3,15 +3,33 @@
     .col-md-3.col-lg-2
       h3 USERS
     .col-md-9.col-lg-10
-      table.table.table-striped
-        thead
-          tr
-            th name
-            th email
-            th role
-        tbody
-          tr
-          -@users.each do |user|
-            td = user.username
-            td = user.email
-            td = user.role
+      - unless @users.length < 1
```

Comment:
It's bad practise to use `unless ... else`. Please use `if @users.any? ... else`.